.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Continuous publication
+on:
+  push:
+    branches: ['**']
+    tags: [v*]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # fetch all tags, required for sbt-dynver
+      - uses: olafurpg/setup-scala@v10
+      - run: sbt ci-release
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+  publishSite:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v10
+      - name: Generate site
+        run: sbt makeSite
+      - uses: JamesIves/github-pages-deploy-action@4.1.3
```

Comment:
Interesting, so we don’t need sbt-ghpages anymore, and we don’t need to create special user credentials as explained here? https://docs.scala-lang.org/overviews/contributors/index.html#continuous-publication

Reply:
Yes that is correct. GitHub exposes tokens to the action that can use it to interact with the repo.

Reply:
Excellent news! Then I guess we can safely remove the sbt-ghpages plugin from this repo?